frontend/packages/dev-console/src/components/import/builder/SelectorCard.tsx

```diff
@@ -4,22 +4,22 @@ import { Card, CardHeader, CardBody } from '@patternfly/react-core';
 import { StarIcon } from '@patternfly/react-icons';
 import { BuilderImage } from '../../../utils/imagestream-utils';
 
-import './BuilderImageCard.scss';
+import './SelectorCard.scss';
 
-export interface BuilderImageCardProps {
+export interface SelectorCardProps {
   image: BuilderImage;
```

Comment:
You should break this prop down into individual props:
* title
* icon
* value (used to notify in onChange)